### PR TITLE
Port: ImageMagick

### PR DESCRIPTION
**Pull request by japetrus**
Initial ImageMagick port with support for png, tiff and jpeg.

### CODE REVIEW

**Comment by awesomekling** [comment]
```
AvailablePorts.md is missing the following ports:
    imagemagick
```

File: Ports/AvailablePorts.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`halflife`](halflife/)                | Half-Life                                                       | 1.0.0                    | https://github.com/SerenityPorts/xash3d-fwgs                                   |
 | [`harfbuzz`](harfbuzz/)                | HarfBuzz                                                        | 2.8.1                    | https://github.com/harfbuzz/harfbuzz                                           |
 | [`hatari`](hatari/)                    | Atari ST/STE/TT/Falcon emulator                                 | 2.4.0-devel              | https://hatari.tuxfamily.org/                                                  |
+| [`imagemagick`](imagemagick/)          | ImageMagick                                                     | 7.1.0-23                 | https://imagemagick.org                                                        |
```
Comment (trflynn89) [inline Ports/AvailablePorts.md:71]: This should be squashed into the previous commit

Comment (japetrus) [reply Ports/AvailablePorts.md:71]: Thank you for telling me. I've done that now.

File: Ports/imagemagick/package.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+version=7.1.0-23
+workdir="${port}-${version}"
+useconfigure="true"
+files="https://github.com/ImageMagick/ImageMagick/archive/refs/tags/${version}.tar.gz imagemagick-v${version}.tar.gz 62c24362891d0af2be9a81d01117195ba0ec8e6982c7568195a33019bfc82188"
+auth_type=sha256
+
+depends=("libpng" "libtiff" "libjpeg")
+
+configure() {
+  run ./configure --host="i686"
```
Comment (linusg) [inline Ports/imagemagick/package.sh:12]: Please don't hardcode this, use `$SERENITY_ARCH`.

**Review by linusg — COMMENTED** [review]
The commit message needs fixing, something like `Ports: Add ImageMagick`. The one from the squashed commit is not relevant.

**Pull request merged**